supabase/migrations/20230207013753_update_types.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE public.users 
+ALTER COLUMN created_at TYPE timestamp without time zone using created_at::timestamp;
+
+ALTER TABLE public.users
+ALTER COLUMN updated_at TYPE timestamp without time zone using updated_at::timestamp;
+
+ALTER TABLE public.issues
+ALTER COLUMN created_at TYPE timestamp without time zone using created_at::timestamp;
+
+ALTER TABLE public.issues
+ALTER COLUMN updated_at TYPE timestamp without time zone using updated_at::timestamp;
+
+ALTER TABLE public.issues
+ALTER COLUMN started_at TYPE timestamp without time zone using started_at::timestamp;
+
+ALTER TABLE public.issues
+ALTER COLUMN completed_at TYPE timestamp without time zone using completed_at::timestamp;
+
+ALTER TABLE public.issues
+ALTER COLUMN closed_at TYPE timestamp without time zone using closed_at::timestamp;
+
+ALTER TABLE public.wallets
+ALTER COLUMN created_at TYPE timestamp without time zone using created_at::timestamp;
+
+ALTER TABLE public.wallets
+ALTER COLUMN updated_at TYPE timestamp without time zone using updated_at::timestamp;
```

Comment:
Can you explain to me these migration files? Are they just for reference? Is this considered the correct way to include the database setup in the codebase? It seems like a good idea but this is just new for me to see.

Reply:
> Can you explain to me these migration files? Are they just for reference? Is this considered the correct way to include the database setup in the codebase? It seems like a good idea but this is just new for me to see.

It is a normal process to have a migration file for every change in database management.